clouds_auth.yaml.template

```diff
@@ -0,0 +1,13 @@
+clouds:
```

Comment:
Can we add metal3-bootstrap to this file also?

Reply:
Actually scratch that, stupid idea

Reply:
@derekhiggins I was wondering about it, any ideas if the bootstrap is deployed using `http_basic` or `noauth` ?

Reply:
Its noath but the reason I said its a stupid idea is because its gone by the time this runs....